QueueInteropTransport.php

```diff
@@ -145,11 +155,39 @@ public function stop(): void
         $this->shouldStop = true;
     }
 
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        $resolver->setDefaults(array(
+            'receiveTimeout' => null,
+            'deliveryDelay' => null,
+            'delayStrategy' => RabbitMqDelayPluginDelayStrategy::class,
+            'priority' => null,
+            'timeToLive' => null,
+            'topic' => array('name' => 'messages'),
+            'queue' => array('name' => 'messages'),
+        ));
+
+        $resolver->setAllowedTypes('receiveTimeout', array('null', 'int'));
+        $resolver->setAllowedTypes('deliveryDelay', array('null', 'int'));
+        $resolver->setAllowedTypes('priority', array('null', 'int'));
+        $resolver->setAllowedTypes('timeToLive', array('null', 'int'));
+        $resolver->setAllowedTypes('delayStrategy', array('null', 'string'));
+
+        $resolver->setAllowedValues('delayStrategy', array(
+                null,
```

Comment:
Style 👮 😄 (i.e. indentations)

Reply:
(I suggest you run `php-cs-fixer fix` on this file).

Reply:
ok

Reply:
Maybe the fixer is configured incorrectly, because the check was ok:
https://travis-ci.org/php-enqueue/messenger-adapter/builds/387351530?utm_source=github_status&utm_medium=notification